.github/workflows/jsdoc-publish.yml

```diff
@@ -44,7 +44,9 @@ jobs:
       - name: get-npm-version
         id: package-version
         uses: martinbeentjes/npm-get-version-action@master
+      # --no-mime-magic --guess-mime-type flags to work around CSS incorrect MIME type issue
+      # (https://stackoverflow.com/questions/53708938/s3cmd-flagging-css-with-wrong-mime-type)
       - name: Upload versioned docs (for archival)
-        run: s3cmd put --recursive --acl-public docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/js/${{ steps.package-version.outputs.current-version}}/
+        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/js/${{ steps.package-version.outputs.current-version}}/
```

Comment:
I was going off the first answer which seemed simpler, if it works (I saw a few other sources mention this, with varying success): https://stackoverflow.com/a/61254294/17700221

If it doesn't work then I'll try the more explicit way

Reply:
I'm not sure what mime-magic is, but it would seem that guess-mime-type would use the file extension to set the mime (and in this command, mime-magic excluded).  That seems safe to me.

Reply:
Yeah, `mime-magic` is some Python script which looks at the contents of the file to guess the MIME type, and seems to get tripped up by CSS. `guess-mime-type` apparently uses the extension which I think should be fine for our use case. Sadly can't test this without merging as I don't have the access keys to run it locally!

Reply:
well...at least we can't make it worse :D